components/docs-chef-io/content/automate/chef_infra_server.md

```diff
@@ -38,6 +38,8 @@ The objects that you can manage from the Chef Infra Server are:
 
 ## Connect Chef Infra Servers to Chef Automate
 
+{{< note >}} Check the article [Adding the Chef Infra Server view to Chef Automate UI](https://community.progress.com/s/article/Adding-Chef-Infra-server-view-to-Chef-Automate-UI) to see the method to automate the process of adding the Chef Infra Server view to Chef Automate UI. {{< /note >}}
```

Comment:
The order of this sentence should be reversed. Tell them first what the outcome of an action is before telling them what action to take.

> To see a method for automating the process of adding Chef Infra Server to Chef Automate, see this support article.

 See here for more details --> https://developers.google.com/style/sentence-structure